sr_wear_logger/src/sr_wear_logger.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+#
+# Copyright <Year> Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+import rospy
+import rospkg
+import os
+import yaml
+import roslaunch
+import requests
+import shutil
+from sensor_msgs.msg import JointState
+from sr_utilities_common.awsmanager import AWS_Manager
+
+
+class WearLogger():
+
+    def _extractHandData(self, msg):
```

Comment:
use right convetion for function and variables names (in all of the files)

Reply:
Checked and verified